src/interfaces/SpokePool.ts

```diff
@@ -51,16 +50,20 @@ export interface V2DepositWithBlock extends V2Deposit, SortableEvent {
 export interface V3Deposit extends V3RelayData {
   destinationChainId: number;
   quoteTimestamp: number;
-  updatedOutputAmount?: BigNumber;
+  realizedLpFeePct?: BigNumber;
```

Comment:
For this commit to go in, we'll leave realizedLpFeePct defined here. A follow-up change will remove it, and the consumers of the realizedLpFeePct on v3 events will have to compute it as needed.